packages/react/package.json

```diff
@@ -21,12 +21,12 @@
     },
     "./web": {
       "import": {
-        "types": "./dist/esm/web.d.ts",
-        "default": "./dist/esm/web.js"
+        "types": "./dist/esm/web/index.d.ts",
+        "default": "./dist/esm/web/index.js"
       },
       "require": {
-        "types": "./dist/cjs/web.d.ts",
-        "default": "./dist/cjs/web.js"
+        "types": "./dist/cjs/web/index.d.ts",
+        "default": "./dist/cjs/web/index.js"
```

Comment:
so this is the actual fix only?

Reply:
Fixed here: https://github.com/lens-protocol/lens-sdk/pull/51/commits/42307333ca23343f5dd642871a1c12c5f3cc1d8f